indra/llaudio/llaudiodecodemgr.cpp

```diff
@@ -70,22 +70,22 @@ class LLVorbisDecodeState : public LLThreadSafeRefCount
 	
 	LLVorbisDecodeState(const LLUUID &uuid, const std::string &out_filename);
 
-	BOOL initDecode();
-	BOOL decodeSection(); // Return TRUE if done.
-	BOOL finishDecode();
+	bool initDecode();
+	bool decodeSection(); // Return true if done.
+	bool finishDecode();
 
 	void flushBadFile();
 
 	void ioComplete(S32 bytes)			{ mBytesRead = bytes; }
-	BOOL isValid() const				{ return mValid; }
-	BOOL isDone() const					{ return mDone; }
+	bool isValid() const				{ return mValid; }
+	bool isDone() const					{ return mDone; }
```

Comment:
<img src="https://media0.giphy.com/media/Od0QRnzwRBYmDU3eEO/giphy.gif"/>

Reply:
Looks good to me